solana/programs/example-native-token-transfers/src/instructions/release_inbound.rs

```diff
@@ -135,25 +173,19 @@ pub fn release_inbound_unlock(
         }
     }
 
-    assert!(inbox_item.release_status == ReleaseStatus::Released);
-    match ctx.accounts.common.config.mode {
-        Mode::Burning => Err(NTTError::InvalidMode.into()),
```

Comment:
Note to self/other reviews: 
Deleting this is OK because it's guaranteed by the new constraint on the composite account `ReleaseInboundUnlock`